config/default.go

```diff
@@ -64,6 +64,8 @@ inDocker: false
 cmdType: "native"
 `
 
+var InternalConfigFields = []string{"generateGithubActions", "keployNetwork", "keployContainer", "inCi", "cmdType"}
+
 var config = &Config{}
```

Comment:
why make it global when we have a function to fetch it as a string? You can make this a local variable.

Reply:
Done.